docs/content/release_notes/limitations.md

```diff
@@ -13,7 +13,7 @@ The following limitations apply when using volume snapshots with the IBM block s
 -   When deleting a PersistentVolumeClaim (PVC), the persistent volume (PV) remains until all snapshots of the specific PV are deleted.
 -   When using the CSI (Container Storage Interface) driver with IBM Spectrum® Virtualize family products, a snapshot can only be used to provision a new volume of equal size.
 
-**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [HyperSwap usage limitations](#hyperSwap-usage-limitations).
+**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [High availability (HA) limitations](#high-availability-ha-limitations).
```

Comment:
does IBM Docs support "nested" anchor links?
```suggestion
**Note:** For volume snapshot limitations pertaining specifically to HyperSwap usage, see [High availability (HA) limitations](#hyperswap-volumes).
```

Reply:
i'll try it out and update here the outcome

Reply:
it seems to work. I've updated files accordingly. if there are places that I missed, let me know.